### PR TITLE
Cds dojson cleaning

### PR DESCRIPTION
**Pull request by kprzerwa**
real change in commit: 
https://github.com/CERNDocumentServer/cds-ils/pull/235/commits/24bd61d51320e7c14c24652d77f91936f07ff1f3

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -20,7 +20,7 @@ universal = 1
 add_ignore = D401
 
 [pycodestyle]
-exclude = docs/conf.py
+exclude = docs/conf.py, tests/migrator/*
```
Comment (kprzerwa) [inline setup.cfg:23]: I had to do this, there is a lot of compared string types which are not getting handled by black

File: setup.py [hunk 2]
```diff
@@ -42,9 +42,7 @@
     "invenio-app-ils[lorem,elasticsearch7,postgresql]==1.0.0a20",
     "sentry-sdk>=0.10.2",
     # migrator deps
-    "invenio-migrator==1.0.0a10",
-    "invenio-records-files",
-    "cds-dojson@git+https://github.com/CERNDocumentServer/cds-dojson@books",
+    "cds-dojson==0.9.0",
```
Comment (kprzerwa) [inline setup.py:45]: invenio-migrator and topical branch of do-json removed

File: cds_ils/importer/providers/cds/rules/utils.py [hunk 3]
```diff
@@ -122,6 +126,186 @@ def extract_volume_info(value):
...
+    d = date(year, 1, 1)
+    if d.weekday() > 3:
+        d = d + timedelta(7 - d.weekday())
+    else:
+        d = d - timedelta(d.weekday())
+    dlt = timedelta(days=(week - 1) * 7)
+    return d + dlt
+
+
+def replace_in_result(phrase, replace_with, key=None):
```
Comment (ntarocco) [inline cds_ils/importer/providers/cds/rules/utils.py:239]: 😱 👀

File: cds_ils/migrator/RecordLoader.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        return record
+
+    @classmethod
+    def create_record(
+        cls, dump, model, pid_provider, legacy_id_key="legacy_recid"
+    ):
+        """Create a new record from dump."""
+        import ipdb
+
+        ipdb.set_trace()
```
Comment (ntarocco) [inline cds_ils/migrator/RecordLoader.py:39]: 👀

**Pull request merged**